src/Network/Network/help/Get-AzNetworkManagerDeploymentStatusList.md

```diff
@@ -0,0 +1,160 @@
+---
+external help file: Microsoft.Azure.PowerShell.Cmdlets.Network.dll-Help.xml
+Module Name: Az.Network
+online version:
+schema: 2.0.0
+---
+
+# Get-AzNetworkManagerDeploymentStatusList
```

Comment:
I would prefer to keep as-is, as this command will always return a list of all deployment statuses. Let me know what you think; it will never return a non-list response.